fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LookupDimension.java

```diff
@@ -23,6 +23,12 @@
 
 /**
  * LookupDimension creates a Look up dimension based on the namespace chain.
+ * <p>
+ * {@link LookupDimension} is essentially a special case of {@link RegisteredLookupDimension} using namespace. Hence
+ * this class ONLY applies to the Druid namespace lookup serialization. See
+ * http://druid.io/docs/latest/querying/dimensionspecs.html#lookup-extraction-function for more details.
+ * <p>
+ * TODO - Change this class to be a subclass of {@link RegisteredLookupDimension}.
```

Comment:
The thing we want to think about is whether we ever need to allow the 'namespace' serialization with features that subclass this behavior.  Possibly the issue is for richer-featured implementation dimensions to use encapsulation rather than subclassing.

Not a problem to be solved in this PR, however.  It probably depends on a simplification via refactor of the dimension interface.

Reply:
@michael-mclawhorn Thanks. I've put this in an issue https://github.com/yahoo/fili/issues/656